utils/admin_console_index.tsx

```diff
@@ -1,12 +1,34 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
-
 import FlexSearch from 'flexsearch/dist/flexsearch.es5';
+import {IntlShape} from 'react-intl';
+
+import {PluginRedux} from '@mattermost/types/plugins';
+
+import AdminDefinition from 'components/admin_console/admin_definition';
```

Comment:
General comment on this file. I am not happy with type definitions as I am missing types for for `AdminDefinition`. Once `AdminDefinition` type is defined this file's types could be further improved